lib/artirix_data_models/gateways/data_gateway.rb

```diff
@@ -258,18 +258,21 @@ def connection_by_config_key(config_key, **others)
         connection config: ArtirixDataModels.configuration.send(config_key), **others
       end
 
-      def connection(config: {}, url: nil, login: nil, password: nil, bearer_token: nil, token_hash: nil)
-        url          ||= config.try :url
-        login        ||= config.try :login
-        password     ||= config.try :password
-        bearer_token ||= config.try :bearer_token
-        token_hash   ||= config.try :token_hash
+      def connection(config: {}, url: nil, login: nil, password: nil, bearer_token: nil, token_hash: nil, log_body_request: false, log_body_response: false)
+        url               ||= config.try :url
+        login             ||= config.try :login
+        password          ||= config.try :password
+        bearer_token      ||= config.try :bearer_token
+        token_hash        ||= config.try :token_hash
+        log_body_request  ||= config.try :log_body_request
```

Comment:
problem: these are boolean configs so if you do that you won't be able to overwrite with a false, it will always go and try to find from config.

I suggest you do a `log_body_request: nil` in the method declaration and then try to access from config only if `nil?`